model_logging/migrations/0001_initial.py

```diff
@@ -21,7 +20,7 @@ class Migration(migrations.Migration):
                 ('date_created', models.DateTimeField(default=django.utils.timezone.now)),
                 ('operation', models.CharField(max_length=255, choices=[('added', 'Added'), ('removed', 'Removed'), ('modified', 'Modified')])),
                 ('model_path', models.CharField(max_length=255)),
-                ('data', pgcrypto.fields.TextPGPPublicKeyField(default='')),
```

Comment:
Instead of changing the existing migration, wouldn't it be better to make a couple of new migrations? ie:

1. Add new unencrypted data field,
1. Populate unencrypted data field with data from encrypted field (and the reverse),
1. Drop old field.